backbone/src/main/res/layout/rsb_form_step_layout.xml

```diff
@@ -13,7 +13,6 @@
         android:layout_height="wrap_content"
         android:paddingBottom="@dimen/rsb_reviewStep_item_bottom_margin_form"
         android:layout_marginStart="@dimen/rsb_reviewStep_item_margin_form"
-        app:layout_constrainedHeight="true"
```

Comment:
I know this now works fine (I tested it), but, can you do one more test please, because the _value_ of this property is not seen in action here. The question is, does the title still look fine *if there are more steps in ReviewStep* and *you have to scroll* to see them? Do all the "form titles" look fine?

Reply:
Yes I already tested these scenarios, and also tested when a step has no title. all scenarios worked fine